.ci/azure/linux.yml

```diff
@@ -564,6 +567,9 @@ jobs:
   - script: |
       set -e
       python3 -m pip install -r $(LAYER_TESTS_DIR)/requirements.txt
+      # Import 'test_utils' installed in '<package_test>/tests/python/openvino'
+      export LD_LIBRARY_PATH=$(python3 -m site --user-site)/openvino/libs:$LD_LIBRARY_PATH
+      export PYTHONPATH=${INSTALL_TEST_DIR}/python
```

Comment:
```suggestion
      export PYTHONPATH=$(INSTALL_TEST_DIR)/python
```

Reply:
I would prefer do not do that, this is bash syntax and {} uses for variable resolving (especially if it contains unresolved variable yet) $() to execute command and get its result